libs/prelude/Prelude/Nat.idr

```diff
@@ -386,16 +393,15 @@ log2 (S n) = log2NZ (S n) SIsNotZ
 --------------------------------------------------------------------------------
 -- GCD and LCM
 --------------------------------------------------------------------------------
-partial
-gcd : Nat -> Nat -> Nat
+gcd : (a: Nat) -> (b: Nat) -> {auto ok: NotBothZero a b} -> Nat
```

Comment:
Can you please make this `.{auto ok : NotBothZero a b}` (adding a dot in front) instead, so it will be ensured to be erased?